src/plugins/plot/src/plot/MCTPlotController.js

```diff
@@ -249,15 +272,14 @@ define([
     MCTPlotController.prototype.onMouseUp = function ($event) {
         this.stopListening(this.$window, 'mouseup', this.onMouseUp, this);
         this.stopListening(this.$window, 'mousemove', this.trackMousePosition, this);
-        if (this.pan) {
-            this.endPan($event);
-        }
-        if (this.marquee) {
-            this.endMarquee($event);
-        }
-        this.$scope.$apply();
```

Comment:
@nikhilmandlik - removing this from here creates an issue - panning does not end on mouse up. We need to keep that behavior as it was

Reply:
fixed in https://github.com/nasa/openmct/pull/2471/commits/ae0633a89b185f2cd8e55aae7585df84e8eae4f2